src/lib/metrics.ts

```diff
@@ -235,6 +243,15 @@ export default class MetricsMonitor {
                     apiTokens.labels(type).set(value);
                 }
 
+                enabledMetricsBucketsPreviousDay.reset();
+                enabledMetricsBucketsPreviousDay.set(
+                    stats.previousDayMetricsBucketsCount.enabledCount,
+                );
+                variantMetricsBucketsPreviousDay.reset();
+                variantMetricsBucketsPreviousDay.set(
+                    stats.previousDayMetricsBucketsCount.variantCount,
+                );
```

Comment:
Does this need to be reset and set?

Reply:
I though that's what we do for all the other metrics. We need to update this one once per day. Other resets are not necessary. On the other hand it's just a count call under the hood.